src/App.js

```diff
@@ -1,11 +1,10 @@
 import "./App.css";
-import { Product } from "./pages/pagesIndex";
-
+import { Wishlist } from "./pages/pagesIndex";
```

Comment:
No need to name index file as pagesIndex, instead just name it as index, so while importing
you only have to give path till "./pages"